doc/role-icingaweb2/module-graphite.md

```diff
@@ -0,0 +1,77 @@
+## Module Graphite
+
+This module only configures the module in Icinga Web and not the graphite backend itself. 
+
+## Variables
+
+### Module Configuration File
+
+The general module parameter like `enabled` and `source` can be applied here.
+
+For the **config** file, create a dictionary with sections as keys and the parameters as values.
+
+Example:
+
+```
+icingaweb2_modules:
+  graphite:
+    enabled: true
+    source: package
+    config:
+      graphite:  # This is the section
+        url: 127.0.0.1:9000
+        user: graphite
+        password: graphitepw
+      ui:
+        default_time_range: 6
+
+```
+
+#### Section graphite
+
+| Variable | Type   | Example          |
+|----------|--------|------------------|
+| url      | string | "127.0.0.1:9000" |
+| user     | string | "graphite"       |
+| password | string | "password"       |
+| insecure | number | "1"              |
+
+#### Section ui
+
+| Variable                | Type   | Example |
+|-------------------------|--------|---------|
+| default_time_range      | number | "1"     |
+| defautl_time_range_unit | string | "hours" |
+| disable_no_graphs_found | bool   | "0"/"1" |
```

Comment:
`insecure` and `disable_no_graphs_found` should **both** either be `number` (or `int`) or `bool`, for consistency since both are essentially toggles.